client/java-armeria/src/test/resources/centraldogma-profile-alice.properties

```diff
@@ -14,5 +14,5 @@
 # under the License.
 #
 
-centraldogma.hosts.0=alice.com
-centraldogma.hosts.1=bob.com:8080
+centraldogma.hosts.0=1.2.3.4.xip.io
```

Comment:
Don't we need to change the name of this file? :)

Reply:
Sure, why not? :-D